src/cli.ts

```diff
@@ -117,10 +118,58 @@ function debugCliInfo(version: string, channel: string, env: Env, config: IConfi
   );
 }
 
+class VersionObject {
```

Comment:
Why would this be a class?  Would it be cleaner as an interface that `getIssueVersionObject` could return?

Reply:
Changed to Interface